CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ if(CMAKE_VERSION VERSION_LESS 3.12)
 else()
   cmake_policy(VERSION 3.5.1...3.13.2)
 endif()
+message(STATUS "Using CMake version ${CMAKE_VERSION}")
```

Comment:
"CMake" capitalization inconsistent with toolchain message below..

Reply:
What is correct though?

Reply:
Hmm. Probably "CMake" since that is what they show on their website. So probably the toolchain message() needs to be fixed.

Reply:
Agreed.